pages.th/osx/pbcopy.md

```diff
@@ -0,0 +1,12 @@
+# pbcopy
+
+> คัดลอกข้อมูลจากอินพุตมาตรฐาน (`stdin`) ไปยังคลิปบอร์ด ซึ่งเทียบได้กับการกดปุ่ม Cmd + C บนแป้นพิมพ์.
```

Comment:
`ซึ่งเทียบได้กับการกดปุ่ม Cmd + C บนแป้นพิมพ์`  (This is comparable to hitting Cmd + C on your keyboard.)

Can you add this part to the [English page](https://github.com/tldr-pages/tldr/blob/main/pages/osx/pbcopy.md) too? [As we want pages to be uniform across the original page and translations]

Reply:
Sure, I already added it to the original English page